python3/uproctrace/tool.py

```diff
@@ -87,4 +152,4 @@ def main():
     Parse command line arguments and execute selected action.
     """
     args = parse_args()
-    args.func(args)
+    sys.exit(args.func(args))
```

Comment:
The functions do not return an exit code, so this is `sys.exit(None)`. What is the idea here?

Reply:
None defaults to zero, see help(sys.exit)
```
exit(...)
    exit([status])
    
    Exit the interpreter by raising SystemExit(status).
    If the status is omitted or None, it defaults to zero (i.e., success).
    If the status is an integer, it will be used as the system exit status.
    If it is another kind of object, it will be printed and the system
    exit status will be one (i.e., failure).
```